jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -377,6 +390,86 @@ public T getValue(T object) {
         return actionsColMeta;
     }
 
+    public List<ColumnMeta<T>> renameProcessVariables(ListTable<T> extendedPagedTable, List<ColumnMeta<T>> columnMetas) {
```

Comment:
Can you just rename it to _renameVariables_ as this method is used for both task and process variables?